src/Persistence_matrix/include/gudhi/Fields/Multi_field_operators.h

```diff
@@ -11,7 +11,7 @@
 /**
  * @file Multi_field_operators.h
  * @author Hannah Schreiber, Clément Maria
- * @brief Contains the @ref Multi_field_operators class.
+ * @brief Contains the @ref Gudhi::persistence_fields::Multi_field_operators "Multi_field_operators" class.
```

Comment:
```suggestion
 * @brief Contains the @ref Gudhi::persistence_fields::Multi_field_operators class.
```
This was a suggestion so the doc would only show the short name (while still linking correctly), but since you didn't do it in the other files, we might as well stick to the long name for consistency.

Reply:
Ah yes, I thought about doing like your suggestion first, but then I thought that if someone reads the file description from the file list, it would be better for the namespaces to be explicit.